<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsView</class>
 <widget class="QWidget" name="EventsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="eventSelectWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="eventBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="addEventButton">
        <property name="text">
         <string>Dodaj wydarzenie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeEventButton">
        <property name="text">
         <string>Usun wydarzenie</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="actionSelectWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="actionBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="addActionButton">
        <property name="text">
         <string>Dodaj akcje</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeActionButton">
        <property name="text">
         <string>Usun akcje</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="actionWidget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
